src/backend/distributed/master/master_node_protocol.c

```diff
@@ -217,7 +218,7 @@ master_get_table_ddl_events(PG_FUNCTION_ARGS)
 		oldContext = MemoryContextSwitchTo(functionContext->multi_call_memory_ctx);
 
 		/* allocate DDL statements, and then save position in DDL statements */
-		tableDDLEventList = GetTableDDLEvents(relationId);
+		tableDDLEventList = GetTableDDLEvents(relationId, includeSequenceDefaults);
```

Comment:
Since this public UDF could be called on a relation on master node, I more inclined to keep it this way. As you stated, it won't make much difference when called on a worker shard